lib/util/problem.js

```diff
@@ -206,7 +206,10 @@ const problems = {
 
     // 409.14 (entityViolation) was used in the past, but has been deprecated
 
-    entityVersionConflict: problem(409.15, ({ current, provided }) => `Current version of the Entity is '${current}' and you provided '${provided}'. Please correct the version number or pass '?force=true' in the URL to forcefully update the Entity.`)
+    entityVersionConflict: problem(409.15, ({ current, provided }) => `Current version of the Entity is '${current}' and you provided '${provided}'. Please correct the version number or pass '?force=true' in the URL to forcefully update the Entity.`),
+
+    datasetNameConflict: problem(409.16, ({ current, provided }) => `A dataset named '${current}' exists and you provided '${provided}' with the same name but different capitalization. Please change the capitalization of '${provided}' to match '${current}'.`)
```

Comment:
Remove second part of sentence to make it make more sense for the new Entity List via API modal.